jbpm-db-scripts/src/main/resources/db/ddl-scripts/postgresql/postgresql-springboot-bytea-jbpm-schema.sql

```diff
@@ -72,7 +72,7 @@ create table SessionInfo (id int8 not null, lastModificationDate timestamp, rule
 create table Task (id int8 not null, archived int2, allowedToDelegate varchar(255), description varchar(255), formName varchar(255), name varchar(255), priority int4 not null, subTaskStrategy varchar(255), subject varchar(255), activationTime timestamp, createdOn timestamp, deploymentId varchar(255), documentAccessType int4, documentContentId int8 not null, documentType varchar(255), expirationTime timestamp, faultAccessType int4, faultContentId int8 not null, faultName varchar(255), faultType varchar(255), outputAccessType int4, outputContentId int8 not null, outputType varchar(255), parentId int8 not null, previousStatus int4, processId varchar(255), processInstanceId int8 not null, processSessionId int8 not null, skipable boolean not null, status varchar(255), workItemId int8 not null, taskType varchar(255), OPTLOCK int4, taskInitiator_id varchar(255), actualOwner_id varchar(255), createdBy_id varchar(255), primary key (id));
 create table task_comment (id int8 not null, addedAt timestamp, text text, addedBy_id varchar(255), TaskData_Comments_Id int8, primary key (id));
 create table TaskDef (id int8 not null, name varchar(255), priority int4 not null, primary key (id));
-create table TaskEvent (id int8 not null, correlationKey varchar(255), logTime timestamp, message varchar(255), processInstanceId int8, processType int4, taskId int8, type varchar(255), userId varchar(255), OPTLOCK int4, workItemId int8, primary key (id));
+create table TaskEvent (id int8 not null, correlationKey varchar(255), logTime timestamp, message varchar(255), processInstanceId int8, processType int4, taskId int8, type varchar(255), userId varchar(255), OPTLOCK int4, workItemId int8, currentOwner varchar(255), primary key (id));
```

Comment:
This change is missing for the `postgresql-springboot-jbpm-schema.sql` file. Please add it there as well